release-calendar/src/client/stylesheets/_base.scss

```diff
@@ -1,16 +1,20 @@
 //Channel Colors
-$lts: MidnightBlue;
-$stable: Crimson;
-$perc-beta: DarkOrchid;
-$perc-experimental: MediumSeaGreen;
-$opt-in-beta: PaleVioletRed;
+$lts: rgb(61, 63, 206);
+$stable: rgb(216, 158, 83);
+$perc-beta: rgb(134, 62, 216);
+$perc-experimental: rgb(100, 204, 135);
+$opt-in-beta: rgb(200, 125, 207);
 $opt-in-experimental: SteelBlue;
-$nightly: Tomato;
+$nightly: rgb(209, 36, 36);
+$amp-blue: rgb(33, 150, 243);
+$amp-off-blue: rgba(23, 93, 151, 0.596);
+$amp-yellow: #fff198d0;
+$text-color: #24292e;
```

Comment:
Make these values consistent as much as you can; either use variables, rgb, rbga, or hex

Reply:
This will be automatically fixed when the scss linter is added, but I manually did it for now.

Reply:
It's still inconsistent?

Reply:
no, it should be good!